Extension/src/LanguageServer/extension.ts

```diff
@@ -1162,3 +1162,39 @@ export function UpdateInsidersAccess(): void {
         void vscode.commands.executeCommand("workbench.extensions.installExtension", "ms-vscode.cpptools", { installPreReleaseVersion: true }).then(undefined, logAndReturn.undefined);
     }
 }
+
+export async function PreReleaseCheck(): Promise<void> {
```

Comment:
Could you switch to camel-case for function names (to match the existing style)?

Reply:
Didn't we have a linter rule on this before?